src/simpleweighteddigraph.jl

```diff
@@ -105,13 +117,41 @@ function add_edge!(g::SimpleWeightedDiGraph, e::SimpleWeightedGraphEdge)
     return true
 end
 
-function rem_edge!(g::SimpleWeightedDiGraph, e::SimpleWeightedGraphEdge)
-    has_edge(g, e) || return false
-    U = weighttype(g)
-    @inbounds g.weights[dst(e), src(e)] = zero(U)
+rem_edge!(g::SimpleWeightedDiGraph, e::AbstractEdge) =
+    rem_edge!(g, src(e), dst(e))
+
+function rem_edge!(g::SimpleWeightedDiGraph{T, U}, u::Integer, v::Integer) where {T<:Integer, U<:Real}
```

Comment:
```suggestion
function rem_edge!(g::SimpleWeightedDiGraph, u::Integer, v::Integer)
```
There is probably no reason to restrict these parameters here, when we don't need them in our code.

Reply:
Agreed, especially since unused type parameters have a performance cost

Reply:
Woops, CI failed